### DISCUSSION TITLE
Gradually migrating an existing Symphony based PHP ERP to .Net 8

### DISCUSSION THREAD

**Discussion by SwissMaWi** [peachpiecompiler/peachpie#1131] [General]
Hello Community
Our customer is running his custom made ERP application on PHP with a mySQL database. We want to migrate it to .Net by gradually migrating every component that is either new or touched for a change.
The application is using the Symphony framework and quite a few PHP components for PDF reporting and network printing.
Has anyone done this before?
Can it be done with PeachPie?
Where are the best resources for us to start making a plan?
Any help or hint is greatly appreciated.
Regards, Martin

**Comment by bfistein** [comment]
Hi Martin,
Thanks for your question. This is one of the key use cases we created PeachPie for. The short answer is: we expect a few minor issues or bugs when getting the PDF library to run, but theoretically it should work. It will definitely work if we systematically fix any bugs. 

People have run Symfony on PeachPie before, it can definitely be done.
The best resources are

- docs.peachpie.io
- peachpie.io/blog
- https://www.peachpie.io/usecases#migrate

You might want to check in again a week or two from now, as we are currently revamping the project a bit and some things may change that could make it a bit easier for you to get going.
